doc/source/index.rst

```diff
@@ -73,7 +73,7 @@ Python API to interact with `Ansys Mechanical`_ (FEA software for structural eng
 
         Overview of Ansys Mechanical scripting.
 
-        :bdg-primary-line:`ACT` :bdg-primary-line:`Threading`
+        :bdg-primary-line:`ACT` :bdg-primary-line:`Threading` :bdg-primary-line:`Script Helpers`
```

Comment:
@klmcadams I'm not exactly sure what this line of code does. I think it provides links to major topics of interest? Anyhow, is there a reason we are using title case here in "Script Helpers" rather than using sentence case?